### PR TITLE
Create molten-vk

### PR DESCRIPTION
**Pull request by iMonZ**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [ ] `brew audit --cask {{cask_file}}` is error-free.
- [ ] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew install --cask {{cask_file}}` worked successfully.
- [ ] `brew uninstall --cask {{cask_file}}` worked successfully.

So there is already a formula but currently nobody is able to fix it.

That is why a cask should be present in the mean time.

Can someone help me?

Is it possible to run install_vulkan.py as sudo and setup-env.sh while installing and uninstall.sh for uninstalling?

### CODE REVIEW

File: Casks/molten-vk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "molten-vk" do
```
Comment (reitermarkus) [inline Casks/molten-vk:1]: `vulkan-sdk`, since this is the full SDK, right?

**Comment by reitermarkus** [comment]
I have this in my tap:

https://github.com/reitermarkus/homebrew-tap/commit/c43fd7cbbf041f081b2f8e6f34fc05565c8ceebf

File: Casks/molten-vk (hunk 1 above)
Comment (iMonZ) [reply Casks/molten-vk:1]: oh that's great!
Can you create a PR for the main repo?
(and maybe bump the version to 1.2.162.1)
This would be amazing!

**Comment by iMonZ** [comment]
> I have this in my tap:
> 
> [reitermarkus/homebrew-tap@c43fd7c](https://github.com/reitermarkus/homebrew-tap/commit/c43fd7cbbf041f081b2f8e6f34fc05565c8ceebf)

oh that's great!
Can you create a PR for the main repo?
(and maybe bump the version to 1.2.162.1)
This would be amazing!

**Comment by miccal** [comment]
The cask file is missing the `.rb` file extension, [and as I asked you the last time you submitted a new Cask](https://github.com/Homebrew/homebrew-cask/pull/88180#issuecomment-680801982), **please** read the documentation concerning [how to add a cask](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md) carefully to ensure that _both_ `audit` and `style` pass.

Closing, as this is available as a Formula and and a Cask in @reitermarkus's Tap.

**Comment by iMonZ** [comment]
> The cask file is missing the `.rb` file extension, [and as I asked you the last time you submitted a new Cask](https://github.com/Homebrew/homebrew-cask/pull/88180#issuecomment-680801982), **please** read the documentation concerning [how to add a cask](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md) carefully to ensure that _both_ `audit` and `style` pass.
> 
> Closing, as this is available both as a Formula in `homebrew-core` and as a Cask in @reitermarkus's Tap.

It was a draft pr

**Comment by miccal** [comment]
> It was a draft pr

That makes no difference -- submissions should still pass `audit` and `style` before making a PR.

**Pull request closed without merging**